lib/package.js

```diff
@@ -509,6 +509,12 @@
 
   function loadPackageConfigPath(loader, pkgName, pkgConfigPath) {
     var configLoader = loader.pluginLoader || loader;
+    if (configLoader.execute === false) {
+      var config = configLoader.getConfig();
+      configLoader = new configLoader.constructor();
+      configLoader.config(config);
+      configLoader.execute = true;
+    }
```

Comment:
My concern is that `pluginLoader` is used in multiple places. Are you sure it's OK to assign it here? Maybe cache it as `loader.configLoader`?